java/java.openjdk.project/src/org/netbeans/modules/java/openjdk/jtreg/ActionProviderImpl.java

```diff
@@ -300,12 +305,34 @@ public String[] getExtraMakeTargets() {
                     try {
                         stop.started();
                         Process jtregProcess = new ProcessBuilder(options).start();
-                        BACKGROUND.post(new CopyReaderWriter(new InputStreamReader(jtregProcess.getInputStream()), io.getOut()));
-                        BACKGROUND.post(new CopyReaderWriter(new InputStreamReader(jtregProcess.getErrorStream()), io.getErr()));
+                        StringWriter errorOutput = new StringWriter();
```

Comment:
how about initializing the size of errorOutput to a constant that will be enough most of the time?